### DISCUSSION TITLE
Example usages in PowerShell?

### DISCUSSION THREAD

**Discussion by skint007** [shibayan/Sharprompt#266] [Q&A]
I noticed in the readme it says PowerShell is supported, however I don't see any examples for PowerShell and I'm not sure how to actually use it in PowerShell. I've Used `Add-Type` to load the Sharprompt.dll, but it seems most of the methods wouldn't be PowerShell friendly as I'm not sure how you would work with generics.

Would it be possible to get an example of how to use this in PowerShell? Or am I miss understanding?

**Comment by shibayan** [comment]
PowerShell support in the README means PowerShell as a shell, so it does not support loading Sharprompt in PowerShell. I suppose it is technically possible.

**Comment by skint007** [comment]
I wrote a quick wrapper for Sharprompt to make it easier to use in PS.
In case anyone finds it useful: https://github.com/skint007Labs/PSSharprompt
